pkg/tracing/otel_exporters.go

```diff
@@ -0,0 +1,67 @@
+package tracing
+
+import (
+	"context"
+	"time"
+
+	"github.com/justtrackio/gosoline/pkg/cfg"
+	"github.com/justtrackio/gosoline/pkg/log"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp"
+)
+
+type OtelExporterSettings struct {
+	Endpoint    string        `cfg:"endpoint" default:"localhost:4318"`
+	UrlPath     string        `cfg:"url_path" default:"/v1/traces"`
+	Compression bool          `cfg:"compression" default:"true"`
+	Insecure    bool          `cfg:"insecure" default:"false"`
+	Timeout     time.Duration `cfg:"timeout" default:"10s"`
+	Retry       RetryConfig   `cfg:"retry"`
+}
+
+type RetryConfig struct {
+	Enabled         bool          `cfg:"enabled" default:"false"`
```

Comment:
Why are retries disabled by default? Sounds like having it enabled would be more robust